README.md

```diff
@@ -549,3 +550,64 @@ Small or medium (depends on the choice of implemented lints or schema extensions
 - [GitHub issues describing not-yet-implemented lints](https://github.com/obi1kenobi/cargo-semver-checks/issues?q=is%3Aissue+is%3Aopen+label%3AE-mentor+label%3AA-lint+)
 - [Opportunities to add new schema, enabling new lints](https://github.com/obi1kenobi/cargo-semver-checks/issues/241)
 - [Query engine adapter](https://github.com/obi1kenobi/trustfall-rustdoc-adapter)
+
+### Implement a cryptographic algorithm in Rust
+
+**Description**
+
+The [RustCrypto Project](https://github.com/RustCrypto) maintains pure Rust implementations of
```

Comment:
I like the idea in general, but this description starts to converge a little bit towards "look at this issue tracker and select some issue to work on". This kind of an idea suggestion is explicitly discouraged by Google and can potentially negatively impact on our idea list. On the other hand, I'm aware that some of the other ideas in our list also look a little bit like this, as they are not super-specific.

Still, I wonder if you could make the idea a little bit more specific, at least by removing the notion of implementing other features (and focusing just on a cryptographic algorithm), and listing a few suggested algorithms directly in the description, rather than linking to the issue tracker?